core/evancore.vue.libs/src/components/nav-list/nav-list.scss

```diff
@@ -106,5 +110,10 @@
         }
       }
     }
+
+    button:first-of-type {
+      margin-top: 0;
+      box-shadow: 230px 0px 3px 3px cssVar('bg-level-3'), 10px 3px 3px -3px cssVar('primary');
+    }
```

Comment:
Do I understand it correctly, that you omit the shadow on top for the first entry? If so, that's not what the figma shows?

Reply:
That's true, but necessary for the third level menu. Looks okay for me, but we can discuss it.

Reply:
Would maybe better to add a class for 3rd level submenu instead and only apply first-of-type there?